### DISCUSSION TITLE
Orvibo CM10ZW not correctly recognized

### DISCUSSION THREAD

**Discussion by Arsach** [Koenkk/zigbee2mqtt#12513] [Devices]
Hi all,

I have just included my Orvibo CM10ZW in my network and it is wrongly identified as model RL804QZB.
Here is the internal model number 82c167c95ed746cdbd21d6817f72c593.

I am running version 1.25.1

Regards,
Arsach

**Comment by Koenkk** [comment]
Added by @presslab-us in https://github.com/Koenkk/zigbee-herdsman-converters/pull/1117, can you confirm the `82c167c95ed746cdbd21d6817f72c593` is a `CM10ZW` instead of `RL804QZB`?

**Reply by Arsach** [reply]
Yes the device I have is a CM10ZW (as displayed on the label on the back of the device).

**Reply by presslab-us** [reply]
My unit has the part number RL804QZB.

I'm not sure if they have simply changed the part number or redesigned the product and are using older firmware.  The current web site has CM10ZW shown:
https://www.orvibo.com/en/product/controlbox.html

An older copy of the same web page has RL804QZB shown:
https://web.archive.org/web/20210306054751/https://www.orvibo.com/en/product/controlbox.html

But some of the specifications have also changed.  In any case the model number advertised over Zigbee must still be "RL804QZB" to be detected as this device?
